DigitalLearningSolutions.Web/Views/LearningPortal/Current/MarkAsComplete.cshtml

```diff
@@ -0,0 +1,44 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current
+@using Microsoft.Extensions.Configuration
+@model SetLearningLogItemCompletionDateViewModel
+@inject IConfiguration Configuration
+
+@{
+  ViewData["Title"] = $"Mark {Model.ResourceName} as complete";
+  ViewData["Application"] = "Learning Portal";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/LearningPortal/Current";
+  ViewData["HeaderPathName"] = "My Current Activities";
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  var exampleDate = DateTime.Today;
+}
+
+@section NavMenuItems {
+  <partial name="Shared/_NavMenuItems" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@(new[] { nameof(Model.Day), nameof(Model.Month), nameof(Model.Year) })" />
+    }
+
+    <h1 class="nhsuk-heading-xl word-break">Mark @Model.ResourceName as complete</h1>
+
+    <form class="nhsuk-u-margin-bottom-3" method="post" novalidate>
+      <input type="hidden" asp-for="LearningLogItemId" />
+      <input type="hidden" asp-for="ResourceName" />
+      <vc:date-input id="completion-date"
+                     label="Completion date"
+                     day-id="Day"
+                     month-id="Month"
+                     year-id="Year"
+                     css-class=""
+                     key="1"

```

Comment:
What is this key property?

Reply:
Hmm, I think I copied this from somewhere and didn't notice that. Remove it and nothing has changed